src/codeintel/lib/codeintel2/database/langlib.py

```diff
@@ -1351,6 +1403,9 @@ def cull_mem(self):
                         del self._is_index_dirty_from_dbsubpath[dbsubpath]
                     del self._index_and_atime_from_dbsubpath[dbsubpath]
                 # Note: Dirty indexes that are recently accessed are intentionally not written to disk.
+
+            # Free cached blobs
+            self._cull_cached_blobs(TIME_SINCE_ACCESS)
```

Comment:
I'd switch to the pref for ageout that i requested unless you can think of a reason to keep this as 5 mins.

Reply:
I used a pref set to 60 seconds.  I only used 5 minutes since that's the interval `cull_mem` runs at and used for the other cache.